src/styles/dashboard.slate.css

```diff
@@ -1,5 +1,6 @@
 /* SPDX-License-Identifier: GPL-3.0-or-later */
 :root {
+    --color-main: #FFFFFF;  
```

Comment:
I'm sorry about this, but the `color-[primary|border]` are not used in here, we need to add them in the styles. You can check the light dashboard css for the lines affected.

Reply:
Can you elaborate a bit more on this one? I am probably missing something

Reply:
I am using them in [main.css](https://github.com/netdata/dashboard/blob/15aca8895c7e699c07d94032347aac27ae145901/src/styles/main.css#L95-L103) so that's why I put them here so main.css can access them depending theme